.travis.yml

```diff
@@ -8,3 +8,9 @@ env:
   - STACK=cedar-14
 script:
   - bin/docker-build.sh $STACK
+  - |
```

Comment:
I would actually prefer that it stayed in the Travis file. It's only used there and I'd like to get rid of the top-level bin directory.